src/plays/lofi-player/readme.md

```diff
@@ -0,0 +1,27 @@
+# Lofi Player
+
+Lofi music player using react
+
+## Play Demographic
+
+- Language: js
+- Level: Intermediate
+
+## Creator Information
+
+- User: riyajath-ahamed
+- Gihub Link: https://github.com/riyajath-ahamed
+- Blog: null
+- Video: null
+
+## Implementation Details
+
+Update your implementation idea and details here
```

Comment:
Could you please add more implementation details? As there is no blog, people will be interested in reading it here.